lib/twilio/call.ts

```diff
@@ -863,19 +863,15 @@ class Call extends EventEmitter {
   }
 
   /**
-   * Send a message to Twilio.
-   *
-   * @example
-   * ```ts
-   * call.sendMessage(Call.MessageType.UserDefinedMessage, {
-   *   ahoy: 'world!,
-   * });
-   * ```
-   *
-   * @param messageType - The type of the message to send to Twilio.
-   * @param content - The content of the message to send to Twilio.
+   * TODO
    */
-  sendMessage(messageType: Call.MessageType, content: any) {
+  sendMessage(message: Call.Message): string {
```

Comment:
Does returning a `Promise` here that resolves when we receive `ack` make sense?

Reply:
We initially thought of returning a promise once ack is received. However, it's not the pattern that is used in our SDK. We use pub-sub style for every event that we receive from signaling. In this case, we emit `messageSent` once we receive the `ack`.